subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -2117,20 +2120,26 @@ static void *radio_ccm_ext_rx_pkt_set(struct ccm *cnf, uint8_t phy, uint8_t pdu_
 		CCM_MODE_MACLEN_Msk;
 
 #elif !defined(CONFIG_SOC_SERIES_NRF51X)
```

Comment:
Consider doing 

```c
#if defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
#elif defined(CONFIG_SOC_SERIES_NRF51X)
#else /* !CONFIG_SOC_COMPATIBLE_NRF54LX && !CONFIG_SOC_SERIES_NRF51X */
#endif
```

instead. Having the `else` part in the middle makes it harder to follow.

Reply:
`!defined(CONFIG_SOC_SERIES_NRF51X)`  applies to nRF52 and nRF53. There is no `CCM_MODE_LENGTH_Extended`  in nRF51.

Your suggestion will be then:
```
#if defined(CONFIG_SOC_COMPATIBLE_NRF54LX)
/* some code here */
#elif defined(CONFIG_SOC_SERIES_NRF51X)
/* no code here */
#else /* !CONFIG_SOC_COMPATIBLE_NRF54LX && !CONFIG_SOC_SERIES_NRF51X */
/* some code here */
#endif
```

Reply:
> Your suggestion will be then:

Yeah, so that the "else" clause is at the end (which `!defined(CONFIG_SOC_SERIES_NRF51X)` effectively is here)